pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -202,16 +202,24 @@ func (lww *sqlLastWriteWinsRowProcessor) ProcessRow(
 var (
 	implicitTxnOverrides = sessiondata.InternalExecutorOverride{
 		AttributeToUser: true,
-		// TODO(ssd): we do this for now to prevent the data from being emitted back
-		// to the source. However, I don't think we want to do this in the long run.
-		// Rather, we want to store the inbound cluster ID and store that in a way
-		// that allows us to choose to filter it out from or not. Doing it this way
-		// means that you can't choose to run CDC just from one side and not the
-		// other.
-		DisableChangefeedReplication: true,
+
+		// The OriginIDForLogicalDataReplication session variable will bind the
+		// origin ID 1 to each batch request header sent by the internal executor.
```

Comment:
note to self: i should clarify that there's one batch request header for each sql statment executed by the IE. This is lower level than the batching done by the lww_row_processor